core/src/main/java/hudson/Launcher.java

```diff
@@ -181,6 +181,8 @@ public Computer getComputer() {
          */
         protected boolean reverseStdin, reverseStdout, reverseStderr;
 
+        protected boolean killWhenInterrupted;
```

Comment:
🐛 it causes a regression in external API users. The default value will be `false` though the logic was always killing processes before the change

Reply:
You are right. Flag was inverted.